pkgs/development/compilers/tinygo/default.nix

```diff
@@ -101,12 +101,7 @@ buildGoModule rec {
     sed -i "/GOOS=windows/d" Makefile
     sed -i "/GOOS=darwin/d" Makefile
 
-    # tinygo needs versioned binaries
-    mkdir -p $out/libexec/tinygo
-    ln -s ${lib.getBin clang.cc}/bin/clang $out/libexec/tinygo/clang-${llvmMajor}
-    ln -s ${lib.getBin lld}/bin/ld.lld $out/libexec/tinygo/ld.lld-${llvmMajor}
-    ln -s ${lib.getBin lld}/bin/wasm-ld $out/libexec/tinygo/wasm-ld-${llvmMajor}
-    ln -s ${gdb}/bin/gdb $out/libexec/tinygo/gdb-multiarch
+
```

Comment:
```suggestion
```

Reply:
@SuperSandro2000 will do
can you fix Darwin build :smile:

Reply:
Try including the package including the `stddef.h` header file. Not sure which one that is, you can try finding it via nix-locate.